hastory-api/src/Hastory/API.hs

```diff
@@ -74,6 +75,11 @@ mkHastoryClient url username password = do
         Left err -> pure $ Left err
         Right token -> pure $ Right (HastoryClient clientEnv token)
 
+mkUnauthenticatedClient :: MonadIO m => BaseUrl -> m ClientEnv
+mkUnauthenticatedClient url = liftIO (acceptManager <$> newManager tlsManagerSettings)
```

Comment:
I'm not sure we need a new function for this. Is there a reason you made it? Just calling `newManager` and `mkClientEnv` is what usually happens.
I'm happy to merge but this is a bit silly I think.

Reply:
@NorfairKing  The reason I made this function is because `hastory-api` already depends on `http-client` (and now `http-conduit`), and I didn't want to add those deps to `hastory-cli`. I like it personally, but I'll change it in no time if you say so.

Reply:
There's no problem with also depending on those, that makes no difference :)

Reply:
@NorfairKing No problem. Done!